pkg/networkmanager/firewall.jsx

```diff
@@ -120,6 +121,23 @@ class SearchInput extends React.Component {
     }
 }
 
+const renderPorts = ports => {
+    let tcpPorts = [];
+    let udpPorts = [];
+    for (let port of ports) {
+        if (port.protocol === "tcp")
+            tcpPorts.push(port.port);
+        else
+            udpPorts.push(port.port);
+    }
+    return (
+        <React.Fragment>
+            { tcpPorts.length > 0 && <Row>&nbsp;<strong>TCP: </strong>{ tcpPorts.join(', ') }</Row> }
+            { udpPorts.length > 0 && <Row>&nbsp;<strong>UDP: </strong>{ udpPorts.join(', ') }</Row> }
+        </React.Fragment>
```

Comment:
Can you please add some check for the expected ports to the tests? The ones we add in the tests should have predictable ports across all OSes.

Reply:
https://github.com/cockpit-project/cockpit/blob/43b58346eaa8f9d3f8bd41b9703f455a472a1eca/test/verify/check-networking-firewall#L148 added!